src/plugins/options/option/theme.less

```diff
@@ -1,43 +1,51 @@
 .uip-root {
-  &.uip-light-theme {
-    background-color: @light-theme;
+  background-color: @light-theme;
 
-    & > .uip-plugin {
-      color: @dark-text;
+  & > .uip-plugin {
+    color: @dark-text;
 
-      & > .uip-plugin-inner {
-        background-color: @light-theme-section;
-      }
+    & > .uip-plugin-inner {
+      background-color: @light-theme-section;
+    }
 
-      .snippets-dropdown {
-        &-wrapper {
-          background-color: @light-theme;
-        }
+    &.uip-header .snippets-dropdown {
+      &-wrapper {
+        background-color: @light-theme;
+      }
 
-        &-control {
-          &:before {
-            background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 9' xml:space='preserve'%3E%3Cpolyline points='0,0 8,8 16,0' fill='none' stroke='black' stroke-width='2'%3E%3C/polyline%3E%3C/svg%3E");
-          }
+      &-control {
+        &:before {
+          background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 9' xml:space='preserve'%3E%3Cpolyline points='0,0 8,8 16,0' fill='none' stroke='black' stroke-width='2'%3E%3C/polyline%3E%3C/svg%3E");
         }
       }
     }
   }
 
-  &.uip-dark-theme {
+  &.dark-theme {
     background-color: @dark-theme;
 
     & > .uip-plugin {
       color: @light-text;
 
-      .snippets-dropdown {
-        &-wrapper {
-          background-color: @dark-theme;
-        }
+      & > .uip-plugin-inner {
+        background-color: @dark-theme-section;
+      }
 
-        &-control {
-          &:before {
-            background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 9' xml:space='preserve'%3E%3Cpolyline points='0,0 8,8 16,0' fill='none' stroke='white' stroke-width='2'%3E%3C/polyline%3E%3C/svg%3E");
+      &.uip-header {
```

Comment:
I still see no reason to make header a plugin in out current implementation. here and in other places at least we can use tag selector or at all not clarify placement in header.